chainer/datasets/svhn.py

```diff
@@ -4,7 +4,9 @@
 try:
     from scipy import io
     _scipy_available = True
-except Exception:
+except ImportError:
+    _scipy_available = False
+except AttributeError:
```

Comment:
Since these two exceptions are handled identically, you may want to catch them in a single line. Thanks for this additional change. https://docs.python.org/3/tutorial/errors.html#handling-exceptions

Reply:
I am sorry but I don't have that much spare time.

Reply:
Do you want me to send a PR to your fork?

Reply:
Ok